apps/native-component-list/webpack.config.js

```diff
@@ -0,0 +1,142 @@
+const path = require('path');
+const webpack = require('webpack');
+const pckg = require('./package.json');
+const appDirectory = path.resolve(__dirname, './');
+
+const moduleDirectory = path.resolve(__dirname, '../../node_modules/');
```

Comment:
This seems kind of fragile and coupled to our repo structure. Let's do some more research on how to get webpack projects working with monorepos / Yarn workspaces / Lerna repos.